src/material-experimental/popover-edit/_popover-edit-theme.scss

```diff
@@ -72,6 +72,10 @@
                     opacity 100ms mat.$private-swift-ease-out-timing-function,
                     background-color 300ms mat.$private-swift-ease-out-timing-function;
         visibility: visible;
+
+        @include cdk.high-contrast {
+          border-bottom: 3px dashed black;
```

Comment:
So far I've been avoiding using borders for focus indication since they can make the content jump. Would it make sense to use `outline` here or achieve the same effect with a `position: absolute` pseudo element?

Reply:
And to clarify - this selector is already modifying an absolutely positioned pseudo element.